charts/external-dns/values.yaml

```diff
@@ -21,6 +21,9 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+# Specifies whether the custom resource definition of DNSEndpoint should be deployed
+deployCrd: false
```

Comment:
CRDs shouldn't be installed in a Helm template, there is a CRDs folder for automated CRD installation although you shouldn't use helm to manage CRDs in production.